src/Card/Card.js

```diff
@@ -1,5 +1,5 @@
 const Card = (props) => {
-  const url = `https://raw.githubusercontent.com/Perham94/YGOAnimeMangaHQPics/master/${props.id}.png`;
+  const url = `https://raw.githubusercontent.com/Perham94/YGOAnimeMangaHQPics/master/1000000${props.id}.png`;
```

Comment:
Você pode simplesmente somar o número, que é uma solução melhor do que concatenar strings. Isso inclusive resolve o problema de não carregar as imagen, eu imagino.

```suggestion
  const url = `https://raw.githubusercontent.com/Perham94/YGOAnimeMangaHQPics/master/${100000000 + props.id}.png`;
```

Reply:
Tava olhando isso ontem, no caso é por que alguns números realmente não tem imagem. mas estou pensando em usar outras imagens, das cartas mesmo, mas queria ver contigo a possibilidade de fazer um scraping de uma página só pra pegar os links da imagem, e usar os números como parâmetro, pois eles não seguem uma ordem.

Reply:
Ihhhh, verdade... Por exemplo apenas sorte um número de 20 a 100, e some esse número com `100000000`, que o problema se resolve por agora e teremos 80 cartas pra variar o jogo.

<img width="292" alt="Captura de Tela 2021-11-26 às 11 50 18" src="https://user-images.githubusercontent.com/8717661/143598274-eb631002-d159-44b2-93d2-b1613ef0258c.png">

Reply:
Tem que ser um limite ainda menor fiz alteração ai no próximo commit.

Reply:
De 20 a 100 tem todos os números pelo que eu chequei, então é um _range_ seguro pra sortear

Reply:
Acho que não tem todos, mas posso colocar.